### PR TITLE
Fix/committee request

### PR DESCRIPTION
**Pull request by ariskk**
This PR introduces the following:

- Adds all committees for a slot to the cache to avoid double fetching if the beacon committee service is running behind.
- Remove an obsolete comment

### CODE REVIEW

File: services/blocks/standard/handler.go [hunk 1]
```diff
@@ -743,17 +743,24 @@ func (s *Service) beaconCommittee(ctx context.Context,
...
-			}
-			beaconCommittees[slot][index] = beaconCommittee
-			log.Debug().Uint64("slot", uint64(slot)).Uint64("index", uint64(index)).Msg("Obtained beacon committee from API")
-			return beaconCommittee, nil
+		newBeaconCommittee := &chaindb.BeaconCommittee{
+			Slot:      chainBeaconCommittee.Slot,
+			Index:     chainBeaconCommittee.Index,
+			Committee: chainBeaconCommittee.Validators,
 		}
+		// beaconCommittees cache is slot-scoped, no point adding other slots
```
Comment (mcdee) [inline services/blocks/standard/handler.go:754]: beaconCommittees is shared for all attestations in the block, which could come from different slots, so I think this restriction could result in multiple fetches for a block when only one would be required.  I don't see any harm in removing the conditional here, and storing all committees in the map.

Comment (ariskk) [reply services/blocks/standard/handler.go:754]: This is how it was originally implemented but then I then I added a log line to see the cache hits in action. I realised that this `map` is instantiated in [updateAttestationsForBlock](https://github.com/wealdtech/chaind/blob/master/services/blocks/standard/handler.go#L237) for every blockRoot/slot pair and thus not shared between blocks.
Ideally we would want to fetch once per epoch and cache it for all slots, but that's a bit more involved.

File: services/finalizer/standard/handler.go [hunk 2]
```diff
@@ -484,9 +484,6 @@ func (s *Service) fetchBlock(ctx context.Context, root phase0.Root) (*chaindb.Bl
 			return nil, nil
 		}
 
-		// We need to ensure the finalizer is not running ahead of the blocks service.  To do so, we compare the slot of the block
```
Comment (mcdee) [inline services/finalizer/standard/handler.go:487]: Not sure why this comment has been removed.  It appears to accurately describe the inter-module situation and the purpose of the code following.  Please could you explain your rationale here?

Comment (ariskk) [reply services/finalizer/standard/handler.go:487]: The finalizer was ~2000 slots ahead of the block service (`finalizer.standard` vs `blocks.standard`), which I didn't expect. This has happened a couple of times. 
The code will return `nil` or an error if a) there is a fetching error in `LatestBlocks` b) there are no blocks c) the block version is earlier than the phase it is supposed to be
In all other cases it is execute `OnBlock`. Based on the comment, I was expecting to compare `signedBlock` with `latestBlocks[0]` and return.
I may be missing something?

Comment (mcdee) [reply services/finalizer/standard/handler.go:487]: That sounds like a separate issue.  If the finalizer runs ahead of the blocks module it is possible for the blocks module to reset the finality information, which wouldn't be great.  We should keep the comment and address the issue that caused the finalizer to get ahead.

**Pull request merged**